projects/jpeg-d/index.js

```diff
@@ -13,10 +13,11 @@ async function tvl(timestamp, block, chainBlocks) {
 }
 
 module.exports = {
+  misrepresentedTokens: true,
```

Comment:
@g1nt0ki out of curiosity, why is this set to `true` here?

Reply:
@kargakis we are pricing the nfts in ETH, hence the flag is set as true, this way when tvl changes, we dont mark  it as inflow/outflow of eth token

This is usually set true in cases when we are substituting tokens, say we make an api call to fetch dex tvl, it would be exported as USDT but users are not actually usdt in the protocol

Reply:
Makes sense, thanks! Do you have documentation for all these different knobs anywhere?

Reply:
> Makes sense, thanks! Do you have documentation for all these different knobs anywhere?

thought it was not included in the docs but found it: https://docs.llama.fi/list-your-project/how-to-write-an-sdk-adapter check the module exports section